### DISCUSSION TITLE
How do I change the default port (7860)?

### DISCUSSION THREAD

**Discussion by CalculonPrime** [oobabooga/text-generation-webui#4059] [Q&A] [answered]
You've apparently picked the same default port as Automatic1111's Web UI.

**Comment by missionfloyd** [comment, marked as answer]
[`--listen-port PORT`](https://github.com/oobabooga/text-generation-webui#:~:text=%2D%2Dlisten%2Dport%20LISTEN_PORT)

It's the same because 7860 is Gradio's default port.

**Comment by CalculonPrime** [comment]
For other readers, I found that you need to place this flag in the file called CMD_FLAGS.txt  This is bit different than Automatic1111's Web UI, where you just put command line flags into the startup bat file.

**Reply by DakotaIrsik** [reply]
And don't be ignorant like me and accidentally leave the comment in the CMD_FLAGS.txt wondering why it wouldn't work. does anyone know about changing the hostname from IP? Or an actual piece of documentation for this stuff?
